Makefile

```diff
@@ -7,13 +7,17 @@ all : data/processed/disclosures.xlsx			\
       data/processed/candidate_committee_filings.csv	\
       data/processed/pac_committees.csv			\
       data/processed/pac_committee_filings.csv		\
+			data/intermediate/lobbyists.csv     \
+			data/processed/lobbyist_employer.csv    \
```

Comment:
Weird nitpick, but what's up with the indentation here?

Reply:
My bad-- sometimes my editor adds these spaces. Fixed.